src/store/index.ts

```diff
@@ -63,6 +64,22 @@ const reducers = combineReducers({
   [CURRENT_SESSION_REDUCER_ID]: currentSession,
 })
 
+export type State = {
```

Comment:
oops! https://github.com/gnosis/safe-react/blob/feature/%23934-advanced-settings/src/store/index.ts#L70

Reply:
Same here

Reply:
what about the `CombinedState`? Shall we use it? https://stackoverflow.com/a/58047671/2856529

Reply:
Yes sure! just forgot it, now we won't have merge issues (I hope)

Reply:
:crossed_fingers: